src/cogent3/app/dist.py

```diff
@@ -99,9 +99,9 @@ def _est_dist_pair_slow(self, aln):
         lf.optimise(max_restarts=0, show_progress=False)
         return 2 * lf.get_param_value("length", edge=aln.names[0])
 
-    T = Union[SerialisableType, AlignedSeqsType]
-
-    def main(self, aln: T) -> T:
+    def main(
+        self, aln: Union[SerialisableType, AlignedSeqsType]
```

Comment:
don't add serialisable to any input type unless the app is a writer. So change this to
```python
def main(self, aln: AlignedSeqsType) -> Union[SerialisableType, PairwiseDistanceType]: ...
```